### PR TITLE
Feat:Updated component to support props orientation: horizontal/vertical and label position: top/bottom/left/right wrt step node

### PR DESCRIPTION
**Pull request by ReshmaJoshy**

### CODE REVIEW

**Review by bilta-keyvalue — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -20,11 +20,11 @@
   },
   "repository": {
     "type": "github",
-    "url": "git+https://github.com/KeyValueSoftwareSystems/react-vertical-stepper.git"
+    "url": "git+https://github.com/KeyValueSoftwareSystems/react-stepper.git"
   },
   "author": "Keyvalue",
   "license": "ISC",
```
Comment (arjunrtk-kv) [inline package.json:26]: The License is MIT in LICENSE file, here its ISC. Could you please check.

File: src/node/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+import Node from "./node";
+
+export default Node;
```
Comment (arjunrtk-kv) [inline src/node/index.ts:3]: Please add empty line at the end of file

File: src/node/node.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { FC } from "react";
+import type { INodeProps } from "./types";
+import { Elements } from "../constants";
```
Comment (arjunrtk-kv) [inline src/node/node.tsx:3]: Please be consistant with quotes, use either single or double quotes throughout the code

File: src/node/node.tsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+            alt=""
+          />)
+        || index + 1}
+      </>
+    )}
+    </div>
+  );
+};
+
+export default Node;
```
Comment (arjunrtk-kv) [inline src/node/node.tsx:52]: add empty line at the end

File: src/stepper/stepperComponent.tsx [hunk 5]
```diff
@@ -0,0 +1,268 @@
+import React, { useRef, useEffect, useState } from "react";
```
Comment (arjunrtk-kv) [inline src/stepper/stepperComponent.tsx:1]: can we split this file into multiple for each component used. I feel like the file is pretty long

File: src/utils/getStyles.ts [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+
+const  getStyles = (styles: { [key in Elements]?: IStyleFunction }, element: Elements, step: IStep, index: number): object => {
+  const getElementStyle = styles[element];
+  if (getElementStyle) {
+    return getElementStyle(step, index);
+  }
+  return {};
+};
+
+export default getStyles;
```
Comment (arjunrtk-kv) [inline src/utils/getStyles.ts:13]: please add empty line at the end

**Review by arjunrtk-kv — CHANGES_REQUESTED** [review]

**Review by manacy-keyvalue — APPROVED** [review]

**Pull request merged**